projects/ad9695_fmc/common/ad9695_fmc_bd.tcl

```diff
@@ -0,0 +1,154 @@
+
+# RX parameters
+set RX_NUM_OF_LANES 4      ; # L
+set RX_NUM_OF_CONVERTERS 2 ; # M
+set RX_SAMPLES_PER_FRAME 1 ; # S
+set RX_SAMPLE_WIDTH 16     ; # N/NP
+
+set RX_SAMPLES_PER_CHANNEL 4 ; # L * 32 / (M * N)
+
+source $ad_hdl_dir/library/jesd204/scripts/jesd204.tcl
+
+# adc peripherals
+
+ad_ip_instance axi_adxcvr axi_ad9695_rx_xcvr [list \
+  NUM_OF_LANES $RX_NUM_OF_LANES \
+  QPLL_ENABLE 1 \
+  TX_OR_RX_N 0 \
+  SYS_CLK_SEL 3 \
+  OUT_CLK_SEL 3 \
+  ]
+
+adi_axi_jesd204_rx_create axi_ad9695_rx_jesd $RX_NUM_OF_LANES
+ad_ip_parameter axi_ad9695_rx_jesd/rx CONFIG.SYSREF_IOB false
+
+ad_ip_instance util_cpack2 util_ad9695_rx_cpack [list \
+  NUM_OF_CHANNELS $RX_NUM_OF_CONVERTERS \
+  SAMPLES_PER_CHANNEL $RX_SAMPLES_PER_CHANNEL \
+  SAMPLE_DATA_WIDTH $RX_SAMPLE_WIDTH \
+  ]
+
+adi_tpl_jesd204_rx_create rx_ad9695_tpl_core $RX_NUM_OF_LANES \
+                                               $RX_NUM_OF_CONVERTERS \
+                                               $RX_SAMPLES_PER_FRAME \
+                                               $RX_SAMPLE_WIDTH
+
+ad_ip_instance axi_dmac axi_ad9695_rx_dma [list \
+  DMA_TYPE_SRC 2 \
+  DMA_TYPE_DEST 0 \
+  CYCLIC 0 \
+  SYNC_TRANSFER_START 1 \
+  DMA_2D_TRANSFER 0 \
+  DMA_DATA_WIDTH_SRC [expr 32*$RX_NUM_OF_LANES] \
+  MAX_BYTES_PER_BURST 256 \
+  AXI_SLICE_DEST false \
+  AXI_SLICE_SRC false \
+  DMA_DATA_WIDTH_DEST 128 \
+  FIFO_SIZE 32 \
+  ]
+
+# common cores
+# fPLLClkin = 200 MHz => RX_CLK25_DIV = 8
```

Comment:
Please update the comment to reflect the 325MHz ref clock.

Reply:
updated